### PR TITLE
Fix for #600

### PR DESCRIPTION
**Pull request by newsiberian**
Hi, quick fix for #600

### CODE REVIEW

File: packages/reaction-core/client/main.js [hunk 1]
```diff
@@ -27,8 +27,11 @@ _.extend(ReactionCore, {
           typeof Locale.currency === "object" &&
           typeof Locale.locale.currency === "string") {
           const localeCurrency = Locale.locale.currency.split(",")[0];
-          Locale.currency.rate = shop.currencies[localeCurrency].rate;
-          localeDep.changed();
+          if (shop.currencies[localeCurrency] &&
+            typeof shop.currencies[localeCurrency].rate === "number") {
```
Comment (aaronjudd) [inline packages/reaction-core/client/main.js]: Won't this fail if `shop.currencies[localeCurrency]` is not defined?

**Comment by newsiberian** [comment]
must now work fine

**Comment by newsiberian** [comment]
Hi, as far as I know in such syntax `typeof shop.currencies[localeCurrency] === "object" &&  typeof shop.currencies[localeCurrency].rate === "number"` the second part will not begins to be processed if the first part fails, but if you like `if-if` syntax more, I could update it)

**Comment by aaronjudd** [comment]
OK this is good, I'm not sure why I was seeing error with the original. @newsiberian you're right - it should have worked. I just took the code out and tested and it was fine.  
Approved!

**Pull request merged**